### PR TITLE
fix create credentials command

### PR DESCRIPTION
**Pull request by MariyaSin**
http://outsourcing.nat.tepkom.ru:20080/redmine/issues/673

### CODE REVIEW

**Review by TIove — APPROVED** [review]

File: src/UserService.Business/Commands/Credentials/CreateCredentialsCommand.cs [hunk 1]
```diff
@@ -51,26 +50,30 @@ public CredentialsResponse Execute(CreateCredentialsRequest request)
                 throw new NotFoundException($"Pending user with ID '{request.UserId}' was not found.");
             }
 
+                _userCredentialsRepository.CheckLogin(request.Login, request.UserId);
```
Comment (BelehovEgor) [inline src/UserService.Business/Commands/Credentials/CreateCredentialsCommand.cs:53]: Extra spaces

**Review by BelehovEgor — APPROVED** [review]

File: src/UserService.Business/Commands/Credentials/CreateCredentialsCommand.cs (hunk 1 above)
Comment (MariyaSin) [reply src/UserService.Business/Commands/Credentials/CreateCredentialsCommand.cs]: done

File: src/UserService.Business/Commands/Credentials/CreateCredentialsCommand.cs [hunk 2]
```diff
@@ -51,26 +50,30 @@ public CredentialsResponse Execute(CreateCredentialsRequest request)
                 throw new NotFoundException($"Pending user with ID '{request.UserId}' was not found.");
             }
 
+            _userCredentialsRepository.CheckLogin(request.Login, request.UserId);
+
             if (request.Password != dbPendingUser.Password)
             {
                 throw new ForbiddenException();
```
Comment (djino97) [inline src/UserService.Business/Commands/Credentials/CreateCredentialsCommand.cs:57]: add message exception

File: src/UserService.Business/Commands/Credentials/CreateCredentialsCommand.cs [hunk 3]
```diff
@@ -90,7 +93,7 @@ public CredentialsResponse Execute(CreateCredentialsRequest request)
                 _logger.LogError(exc, "Something went wrong while we were creating the user credentials.");
             }
 
-            throw new InvalidOperationException();
+            throw new BadRequestException();
```
Comment (djino97) [inline src/UserService.Business/Commands/Credentials/CreateCredentialsCommand.cs:96]: add exception message

**Review by KostinP — APPROVED** [review]

File: src/UserService.Business/Commands/Credentials/CreateCredentialsCommand.cs (hunk 2 above)
Comment (MariyaSin) [reply src/UserService.Business/Commands/Credentials/CreateCredentialsCommand.cs]: done

File: src/UserService.Business/Commands/Credentials/CreateCredentialsCommand.cs (hunk 3 above)
Comment (MariyaSin) [reply src/UserService.Business/Commands/Credentials/CreateCredentialsCommand.cs]: done

**Pull request merged**